src/predict_request_validation_utils.cpp

```diff
@@ -130,7 +130,10 @@ class RequestValidator {
     Status validateTensorContent(const InputTensorType& proto, ovms::Precision expectedPrecision, size_t bufferId) const;
     Status validateNumberOfShapeDimensions(const ovms::TensorInfo& inputInfo, const InputTensorType& proto) const;
     Status validatePrecision(const ovms::TensorInfo& inputInfo, const InputTensorType& proto) const;
-    bool checkIfNativeFileFormatUsed(const InputTensorType& proto, const std::string inputName) const;
+    bool checkIfNativeFileFormatUsed(const InputTensorType& proto, const std::string inputName, std::shared_ptr<TensorInfo> inputInfo) const;
```

Comment:
We can avoid copying shared_ptr by accepting `TensorInfo&` and passing dereference inputInfo to the function. You can look up my PR.